src/mlpack/tests/rbm_network_test.cpp

```diff
@@ -258,4 +258,55 @@ BOOST_AUTO_TEST_CASE(MiscTest)
   BuildVanillaNetwork<arma::Mat<float>>(X, 2);
 }
 
+
+/*
+ * Tests the RBM Callbacks.
```

Comment:
I think the majority of callbacks tests are part of: https://github.com/mlpack/mlpack/blob/master/src/mlpack/tests/callback_test.cpp do you think we should do the same for the ``RBM`` class?

Reply:
Sure, I will do that.